examples/06-verif-manual/vm-009-large_lateral_deflection_of_unequal_stiffness_springs.py

```diff
@@ -0,0 +1,409 @@
+                                entnum=0,
+                                item1="ITEM",
+                                it1num="SENE")
+
+# Prints nodal solution results of the X, Y, and Z structural displacements
+# and vector sum.
+print(mapdl.prnsol("U", "COMP"))
+
+# Get the value of the displacements in X-direction.
+disp_x = mapdl.get_value(entity="NODE",
+                         entnum=2,
+                         item1="U",
+                         it1num="X")
+
+# Get the value of the displacements in Y-direction.
+disp_y = mapdl.get_value(entity="NODE",
+                         entnum=2,
+                         item1="U",
+                         it1num="y")
+
+
+###############################################################################
+# Check Results
+# ~~~~~~~~~~~~~
+# Finally we have the results of the strain energy and
+# displacements in :math:`X` and :math:`Y` directions, which can be compared with
+# expected target values:
+#
+# - Strain energy of the system :math:`U_{\mathrm{(energy)}} = 24.01\,N-cm`.
+# - Displacement in X-direction :math:`U_x = 8.631\,cm`.
+# - Displacement in Y-direction :math:`U_y = 4.533\,cm`.
+#
+# For better representation of the results we can use ``pandas`` dataframe
+# with following settings below:
+
+# Define the names of the rows.
+row_names = ["Strain Energy, N-cm",
+             "Deflection-x , cm",
+             "Deflection-y , cm"]
+
+# Define the names of the columns.
+col_names = ['Target',
+             'Mechanical APDL',
+             'RATIO']
+
+# Define the values of the target results.
+target_res = np.asarray([24.01, 8.631, 4.533])
+
+# Create an array with outputs of the simulations.
+simulation_res = np.asarray([strain_energy, disp_x, disp_y])
+
+# Identifying and filling corresponding columns.
+main_columns = {
+    "Target": target_res,
+    "Mechanical APDL": simulation_res,
+    "Ratio": list(np.divide(simulation_res, target_res))
+}
+
+# Create and fill the output dataframe with pandas.
+df2 = pd.DataFrame(main_columns, index=row_names)
+
+# Apply style settings for the dataframe.
+df2.style.set_table_styles([
+                            {
+                              "selector": "th",
+                              "props": [('font-size', '16px')]
+                            },
+                            {
+                              "selector": "td",
+                              "props": [('font-size', '16px')]
+                            },
+                            {
+                                "selector": "td:hover",
+                                "props": [("background-color", "#FFF8DC")]
+                            }]).set_properties(**
+                                               {
+                                                    "color": "black",
+                                                    "text-align": "center"
+                                               }).format("{:.2f}")
```

Comment:
```suggestion
df2.head()
```

After the new PR in ansys theme, it is better to use ``df.head()`` instead of ``df.style``. Although both should work.

Anyway, as you said, keep these code and modify later if you wish.

Reply:
Thanks, I will modify it later!